library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java

```diff
@@ -154,8 +156,15 @@ public void onError(String errMsg, int errCode) {
         });
     }
 
+    private boolean avaliableCopyMessage() {
+        return !(message.getDeletedAt() != null
+                || message.getStatus() == MessageStatus.FAILED
+                || message.getType().equals(ModelType.message_error)
```

Comment:
perhaps we need to check for ephemeral message too?

Reply:
Yeah, You are right.